config/settings/development.py

```diff
@@ -35,6 +35,7 @@
 CELERY_TASK_ALWAYS_EAGER = False
 CELERY_BROKER_URL = env("CELERY_BROKER_URL", default=env("REDIS_URL", default=None))
 CELERY_RESULT_BACKEND = CELERY_BROKER_URL
+CELERY_CREATE_MISSING_QUEUES = True
```

Comment:
I don't think this is strictly necessary but explicit seems good.

Reply:
Yea, I think it defaults on, but good to be exlicit.